charts/spv-wallet-stack/charts/spv-wallet/templates/configmap.yml

```
@@ -8,91 +8,114 @@ data:
   config.yaml: |-
     auth:
       require_signing: {{ .Values.config.auth.require_signing }}
-      scheme: {{ .Values.config.auth.scheme }} 
+      scheme: {{ .Values.config.auth.scheme }}
       signing_disabled: {{ .Values.config.auth.signing_disabled }}
       admin_key: ""
     cache:
       cluster:
-        coordinator: {{ .Values.config.cache.cluster.coordinator }} 
-        prefix: {{ .Values.config.cache.cluster.prefix }} 
-        redis: {{ .Values.config.cache.cluster.redis }} 
-      engine: {{ .Values.config.cache.engine }} 
+        coordinator: {{ .Values.config.cache.cluster.coordinator }}
+        prefix: {{ .Values.config.cache.cluster.prefix }}
+        redis: {{ .Values.config.cache.cluster.redis }}
+      engine: {{ .Values.config.cache.engine }}
       redis:
-        dependency_mode: {{  .Values.config.cache.redis.dependency_mode }} 
-        max_active_connections: {{  .Values.config.cache.redis.max_active_connections }} 
-        max_connection_lifetime: {{  .Values.config.cache.redis.max_connection_lifetime }} 
-        max_idle_connections: {{  .Values.config.cache.redis.max_idle_connections }} 
-        max_idle_timeout: {{  .Values.config.cache.redis.max_idle_timeout }} 
+        dependency_mode: {{ .Values.config.cache.redis.dependency_mode }}
+        max_active_connections: {{ .Values.config.cache.redis.max_active_connections }}
+        max_connection_lifetime: {{ .Values.config.cache.redis.max_connection_lifetime }}
+        max_idle_connections: {{ .Values.config.cache.redis.max_idle_connections }}
+        max_idle_timeout: {{ .Values.config.cache.redis.max_idle_timeout }}
         url: "redis://{{  .Release.Name}}-redis-spv-wallet-master:6379"
-        use_tls: {{  .Values.config.cache.redis.use_tls }} 
+        use_tls: {{ .Values.config.cache.redis.use_tls }}
     db:
       datastore:
-        debug: {{  .Values.config.db.datastore.debug }} 
-        engine: {{  .Values.config.db.datastore.engine }} 
-        table_prefix: {{  .Values.config.db.datastore.table_prefix }} 
+        debug: {{ .Values.config.db.datastore.debug }}
+        engine: {{ .Values.config.db.datastore.engine }}
+        table_prefix: {{ .Values.config.db.datastore.table_prefix }}
+      {{- if eq .Values.config.db.datastore.engine "postgres" }}
```

the value for engine must be postgresql - currently it always end up in the else statement for postgresql configuration.

```suggestion
      {{- if eq .Values.config.db.datastore.engine "postgresql" }}
```